bot/seasons/evergreen/snakes/snakes_cog.py

```diff
@@ -465,7 +465,7 @@ def predicate(reaction, user):
     @locked()
     async def antidote_command(self, ctx: Context):
         """
-        Antidote - Can you create the antivenom before the patient dies?
+        Antidote - Can you create the antivenom before the patient dies.
```

Comment:
This doesn't really read well without a question mark. How about

```suggestion
        A game where you race to create an antivenom before the patient dies.
```

instead?